hesabu/parser_test.go

```diff
@@ -0,0 +1,62 @@
+package hesabu
+
+import (
+	"github.com/Knetic/govaluate"
+	"testing"
+)
+
+type ParserTest struct {
+	Name     string
+	Input    string
+	Expected string
+}
+
+func TestCleaner(t *testing.T) {
+	parserTests := []ParserTest{
+		{
+			Name:     "Sanity check",
+			Input:    "a + b",
+			Expected: "a + b",
+		},
+		{
+			Name:     "Replace AND",
+			Input:    "a AND b",
+			Expected: "a && b",
+		},
+		{
+			Name:     "Replace OR",
+			Input:    "a OR b",
+			Expected: "a || b",
+		},
+		{
+			Name:     "Leaves <= alone",
+			Input:    "a <= b",
+			Expected: "a <= b",
+		},
+		{
+			Name:     "Leaves == alone",
+			Input:    "a == b",
+			Expected: "a == b",
+		},
+		{
+			Name:     "Replace single = with ==",
+			Input:    "a=b && b     =     c && d = e",
+			Expected: "a==b && b    ==    c && d==e",
```

Comment:
would add case for checking we don't hurt variable names with and and or
eg operator == 1
eg operatand == 1